apps/daffio/src/scss/styles.scss

```diff
@@ -4,6 +4,23 @@
 
 // Imports
 @import 'daff-typography';
+@import 'daff-theme';
+
+@import "../app/content/homepage/component/homepage-theme.scss";
+@import "../app/content/pwa/components/feature-comparison/feature-comparison-theme";
+@import "../app/content/pwa/pages/pwa/pwa-theme";
+@import "../app/core/footer/simple-footer/simple-footer-theme";
+@import "../app/core/footer/sub-footer/sub-footer-theme";
+
+@import "theme";
+
+@include daff-theme($theme);
+
+@include daffio-homepage-theme($theme);
+@include daffio-pwa-theme($theme);
+@include daffio-simple-footer-theme($theme);
+@include daffio-sub-footer-theme($theme);
+@include daffio-feature-comparison-theme($theme);
```

Comment:
I tried deleting lines 9-13 and 19-23 and the associated theming files, and it didn't seem to make a difference to the UI of daffio. What are they for?

Reply:
It removes the theming styles from these pages and components.

Reply:
Including `daffio-pwa-theme($theme)` _removes_ theming styles from the pwa page? I have no idea what's going on

Reply:
For reference "It" => "deleting the lines"